tests/unit/db/bigquery/test_bigquery_loader.py

```diff
@@ -124,6 +125,43 @@ def test_can_load_this_format_true(self,
         mock_load_job_config.assert_called_with(set(), mock_load_plan)
         self.assertEqual(True, out)
 
+    @patch('records_mover.db.bigquery.loader.load_job_config')
+    def test_load_from_fileobj(self, mock_load_job_config):
```

Comment:
```suggestion
    def test_load_from_fileobj_true(self, mock_load_job_config):
```

Reply:
Also, this test is the mockingjay of functions. 😆 No changes needed, just observing.